file/client.go

```diff
@@ -21,12 +23,25 @@ const (
 	defaultPollingInterval = 10 * time.Second
 )
 
+var (
+	inactivityThreshold   = 30 * time.Second
+	reactivationThreshold = 60 * time.Second
```

Comment:
```suggestion
	inactivityThreshold   = 24 * time.Hour
	reactivationThreshold = 60 * time.Second
```
Let's make the default inactivity 24h to be generous and avoid people missing files by mistake.